dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/domain/FilePersistenceStrategy.kt

```diff
@@ -56,6 +56,10 @@ internal open class FilePersistenceStrategy<T : Any>(
         return fileReader
     }
 
+    override fun clearAllData() {
+        fileReader.dropAllBatches()
```

Comment:
don't you need to call `super` in overridden functions?

Reply:
It's not mandatory, and in this case it's overriding an interface (so no parent implementation)